experiments/phase2_distillation/step3_compare_to_teacher.py

```
@@ -0,0 +1,138 @@
+    return row.get("reasoning_content") or ""
+
+
+def _teacher_reasoning(row: dict) -> str:
+    # teacher uses reasoning (Cerebras)
+    return row.get("reasoning") or ""
+
+
+def main() -> int:
+    step3_dir = Path(sys.argv[1]) if len(sys.argv) > 1 else _latest("step3")
+    step2_dir = Path(sys.argv[2]) if len(sys.argv) > 2 else _latest("step2")
+    print(f"# Step 3 vs Step 2 comparison\n")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)